added Task1_var5 (not finished yet)

NikitinKU opened the pull request:
@UsovaMA проверьте, пожалуйста. (Незаконченное задание)

NikitinKU committed:
added Task1_var5 (not finished yet)

UsovaMA commented:
@NikitinKU Отметила, что начало есть. Разбирайся с перегрузками (обрати внимание, что именно они требуются по заданию, а не функция сложения, например).

Должны быть перегружены: =, +, -, >, <, >=, <=, ==, !=

Если не получается разобраться - можно договориться о мини-встрече в zoom, это касается всех заинтересованных студентов.

UsovaMA committed:
add points class example

UsovaMA committed:
add tests

NikitinKU committed:
Task1_var5 (almost done)

У меня два вопроса:
1) я не понимаю, как реализовать перегрузку вывода в моем случае (посмотрите код, пожалуйста). Как мне выводить то, что я вывожу в функции мейн на данном этапе?
2) Нужно ли реализовывать вывод времени по типу: "08:02:58" (у меня сейчас в таком случае выведется "8:2:58") ?

NikitinKU commented:
@UsovaMA проверьте, пожалуйста

У меня два вопроса:
1) я не понимаю, как реализовать перегрузку вывода в моем случае (посмотрите код, пожалуйста). Как мне выводить то, что я вывожу в функции мейн на данном этапе?
2) Нужно ли реализовывать вывод времени по типу: "08:02:58" (у меня сейчас в таком случае выведется "8:2:58") ?

UsovaMA committed:
some examples add

NikitinKU committed:
Merge branch 'UsovaMA:workbranch' into workbranch

UsovaMA committed:
add template

UsovaMA committed:
update utf-8

NikitinKU committed:
Merge branch 'UsovaMA:workbranch' into workbranch

NikitinKU committed:
0) дописал методы

NikitinKU commented:
@UsovaMA 
Пока только дописал методы.
Выделяю функции сейчас.
Наверное до 00:00 подразумевалось сделать полностью задание, но я надеюсь -1 не поставите (
Завтра - послезавтра будет готова.

UsovaMA committed:
fix view

UsovaMA committed:
Merge branch 'main' of https://github.com/UsovaMA/TemplateWithClasses into workbranch

UsovaMA committed:
add cpplint

UsovaMA committed:
Update README.md

UsovaMA committed:
add new materials

Commit:
add some project and deleted extra files

Commit:
fix by cpplint

Commit:
fix extra ; in class (coding style)

Commit:
add errors in coding style

Commit:
fix coding style

UsovaMA committed:
add example with tasks and methods

NikitinKU committed:
Delete Task3_Triangle.vcxproj

NikitinKU committed:
Delete Task3_Triangle.vcxproj.filters

NikitinKU committed:
Delete Task3_Triangle.vcxproj.user

NikitinKU committed:
Delete TemplateWithClasses.sln

NikitinKU committed:
Delete triangleMain.cpp

NikitinKU committed:
Merge branch 'UsovaMA:workbranch' into workbranch

NikitinKU committed:
Delete .github/workflows directory

NikitinKU committed:
Delete solverizer directory

NikitinKU committed:
Create CW_Tasks

NikitinKU committed:
Delete CW_Tasks

NikitinKU committed:
CW_Tasks folder added

NikitinKU committed:
Начальный набор выполнения классной работы.

NikitinKU committed:
Добавлена сортировка, реализовано меню.

Начальный набор выполнения классной работы 07.04.22
Добавлено:
- сортировка (сортировка слиянием - merge sort)
- минимально организовано меню для взаимодействия с пользователем 
(теперь можно посмотреть, как сортируется массив всеми сортировками)

P.S. 
В скором времени добавлю остальные интересные способы, частично из 5 таска 1-го семестра.

NikitinKU committed:
Delete 123

NikitinKU commented:
@UsovaMA 

Начальный набор выполнения классной работы 07.04.22
Добавлено:
- сортировка (сортировка слиянием - merge sort)
- минимально организовано меню для взаимодействия с пользователем 
(теперь можно посмотреть, как сортируется массив всеми сортировками)

P.S. 
В скором времени добавлю остальные интересные штуки для удобства, частично из 5 таска 1-го семестра.

NikitinKU committed:
2nd_Course - Task 1 

3 конструктора
перегрузки (+, =, <<)
метод print()
